### ISSUE TITLE
When the player shoots with a bow, there will be a delay of about 600ms

### ISSUE THREAD

**Issue by 1470429149** [GeyserMC/Geyser#4945]
### Describe the bug

When the player shoots with a bow, there will be a delay of about 600ms

### To Reproduce

 player shoots with a bow
arrow Stutters in the air for 600ms

### Expected behaviour

arrow Should not be stuck in the air for a while

### Screenshots / Videos

![qq_pic_merged_1723022620413](https://github.com/user-attachments/assets/4f63ab6b-5a9e-4a2f-9ecf-14890ad6f7ac)

### Server Version and Plugins

[17:26:05 INFO]: This server is running Paper version git-Paper-496 (MC: 1.20.4) (Implementing API version 1.20.4-R0.1-SNAPSHOT) (Git: 7ac24a1 on ver/1.20.4)

### Geyser Dump

https://dump.geysermc.org/t5IJ6JVb9L2TQtcYCzpiv0SraPYu4nYK

### Geyser Version

[17:26:50 INFO]: 此服务器正在运行 Geyser 版本 2.4.1-b638 (git-master-8d1e9e9) (Java: 1.21, Bedrock: 1.20.80/1.20.81 - 1.21.2/1.21.3)

### Minecraft: Bedrock Edition Device/Version

1.20.81

### Additional Context

null

**Comment by letsgoawaydev** [comment]
Adding on to this, when the arrow lands it does not play the wiggle/shake animation.

**Comment by lolwhatdidigetbanned** [comment]
Use geyserextras and hurricane it might not be fixed or smth so use it
Banner: you can pm me to join my server :] when it’s done and out I’m trying to get testers and stuff

**Comment by letsgoawaydev** [comment]
> Use geyserextras and hurricane it might not be fixed or smth so use it
> 
> Banner: you can pm me to join my server :] when it’s done and out I’m trying to get testers and stuff

The Arrow Delay Fix in GeyserExtras isnt really a fix, its more of a workaround. Also, its not enabled by default, this should be a issue that gets fixed in Geyser.

**Comment by 1470429149** [comment]
Yes, I tried to use GeyserExtras unfortunately it only supports 1.21 server core

**Comment by lolwhatdidigetbanned** [comment]
> Yes, I tried to use GeyserExtras unfortunately it only supports 1.21 server core

Still works on 1.21.1

**Comment by lolwhatdidigetbanned** [comment]
> Yes, this is bad, because I am using 1.20.4

Just get 1.21.1 with viaversion and viabackwards and viarewind 1,20.4 is dead

**Comment by davideakc** [comment]
Have you solved this problem?

**Comment by 1470429149** [comment]
> Have you solved this problem?

bad, no

**Comment by lumirth** [comment]
Anyone have a fix for this? It's not the end of the world but still relatively annoying.

Edit: if someone on the contributor side of things would be willing to explain why this is happening, I'd be down to try to contribute myself.

**Comment by onebeastchris** [comment]
Should be resolved with https://github.com/GeyserMC/Geyser/commit/184fd19313e143193aeb55c29f111212c2da5abd
